pkg/proto/configuration/bb_worker/bb_worker.proto

```diff
@@ -110,6 +111,12 @@ message NativeBuildDirectoryConfiguration {
   // LEAST_RECENTLY_USED can, assuming the cache size is sufficient.
   buildbarn.configuration.eviction.CacheReplacementPolicy
       cache_replacement_policy = 5;
+
+  // Use clonefile(2) to populate build action trees using APFS copy-on-
+  // write clones.  Only useful on Darwin when the build_directory_path
+  // is on an APFS volume.  When not set (default) uses standard hardlinks
+  // as supported by most all posix filesystems.
```

Comment:
s/posix/POSIX/

Maybe also add some explanation as why you'd want to set this option, namely to keep macOS security frameworks happy.